app/httpPresenter.js

```diff
@@ -0,0 +1,21 @@
+class HttpPresenter {
+    constructor (req, res) {
+        this.req = req;
```

Comment:
Can we not use shorten names?
Maybe like... never? =)

Reply:
life is too short to user full names...